docs/pages/2020_Intro_Week/sections/appendix_computer_code_setup.rst

```diff
@@ -0,0 +1,10 @@
+*These appendices consist of optional exercises, and are mentioned in earlier parts of the tutorial.
+Go through them only if you have time.*
+
+.. _2020:appendix_computer_code_setup:
+
+Setting up computers and codes
+==============================
+
+To setup a computer, please refer to the `online documentation <https://aiida-core.readthedocs.io/en/latest/get_started/computers.html#setup-a-computer>`_.
+Instructions to setup a code for a computer can be `found here <https://aiida-core.readthedocs.io/en/latest/get_started/codes.html>`_.
```

Comment:
As above. I think for this one it should be `` :ref:`found here <how-to:codes>` ``